client/reducers/orders.js

```diff
@@ -0,0 +1,11 @@
+const orders = (state = [], action) => {
```

Comment:
Let's rename this file to `currentOrder.js` and rename `state` on this line to `currentOrder` and make it an object instead of an array. Also, instead of `const orders` you can just `export default function (currentOrder = {}, action)`.